### PR TITLE
Add LibMan CLI doc

### PR DESCRIPTION
**Pull request by scottaddie**
Fixes https://github.com/aspnet/Docs/issues/7246

For consistency, this doc was heavily influenced by the .NET Core CLI doc pages. For example: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test?tabs=netcore21

[Internal Review Page](https://review.docs.microsoft.com/en-us/aspnet/core/client-side/libman/libman-cli?view=aspnetcore-2.1&branch=pr-en-us-8260)

### CODE REVIEW

File: aspnetcore/client-side/index.md [hunk 1]
```diff
@@ -3,14 +3,15 @@ title: Client-side development in ASP.NET Core
...
-ms.date: 08/09/2018
+ms.date: 08/22/2018
 uid: client-side/index
 ---
 # Client-side development in ASP.NET Core
 
 - [Use Gulp](xref:client-side/using-gulp)
 - [Use Grunt](xref:client-side/using-grunt)
 - [Use LibMan](xref:client-side/libman/index)
+  - [LibMan CLI](xref:client-side/libman/libman-cli)
```
Comment (guardrex) [inline aspnetcore/client-side/index.md]: We settled on asterisks for lists. I'm cool either way, but I've been using asterisks in lists and converting dashes to asterisks on topic updates.

File: aspnetcore/client-side/libman/libman-cli.md [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+To create a *libman.json* file in an ASP.NET Core project:
+
+* Navigate to the project root.
+* Run the following command:
+
+  ```console
+  libman init
+  ```
+
+* Type the name of the default provider to be used, or press `Enter` to use the default CDNJS provider. Valid values include:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: Recommend avoiding "to be." I think u can just say ...

> Type the name of the default provider, or press `Enter` to use the default CDNJS provider.

Comma optional.

File: aspnetcore/client-side/libman/libman-cli.md [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+
+### Arguments
+
+`LIBRARY`
+
+The name of the library to install. This name may include version number notation (for example, `@1.2.0`).
+
+### Options
+
+The following options can be appended to the `libman install` command:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: > The following options are available for the \`libman install\` command:

File: aspnetcore/client-side/libman/libman-cli.md [hunk 4]
```diff
@@ -0,0 +1,498 @@
...
+
+* `-d|--destination <PATH>`
+
+  The location to install the library. If not specified, the default location is used. If no `defaultDestination` property is specified in *libman.json*, this option is required.
+
+* `--files <FILE>`
+
+  The name of the file to be installed from the specified library. If not specified, all files from the library are installed. Provide one `--files` option per file to be installed.
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: > Specifies the name of the file to install from the specified library.

Can you tell that my mentors only permitted state verbs on :skull: **_pain of death_!** :skull: :smile: ... We could be rebels, but I'm still afraid of Henderson. Trust me on this ... he **will** rise from the grave and come after us. 🏃😨

File: aspnetcore/client-side/libman/libman-cli.md [hunk 5]
```diff
@@ -0,0 +1,498 @@
...
+### Synopsis
+
+```console
+libman restore [--verbosity]
+libman restore [-h|--help]
+```
+
+### Options
+
+The following options can be appended to the `libman restore` command:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: > The following options are available for the \`libman restore\` command:

File: aspnetcore/client-side/libman/libman-cli.md [hunk 6]
```diff
@@ -0,0 +1,498 @@
...
+### Synopsis
+
+```console
+libman clean [--verbosity]
+libman clean [-h|--help]
+```
+
+### Options
+
+The following options can be appended to the `libman clean` command:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: > The following options are available for the \`libman clean\` command:

File: aspnetcore/client-side/libman/libman-cli.md [hunk 7]
```diff
@@ -0,0 +1,498 @@
...
+
+### Arguments
+
+`LIBRARY`
+
+The name of the library to uninstall. This name may include version number notation (for example, `@1.2.0`).
+
+### Options
+
+The following options can be appended to the `libman uninstall` command:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: > The following options are available for the \`libman uninstall\` command:

File: aspnetcore/client-side/libman/libman-cli.md [hunk 8]
```diff
@@ -0,0 +1,498 @@
...
+
+### Arguments
+
+`LIBRARY`
+
+The name of the library to update.
+
+### Options
+
+The following options can be appended to the `libman update` command:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: > The following options are available for the \`libman update\` command:

File: aspnetcore/client-side/libman/libman-cli.md [hunk 9]
```diff
@@ -0,0 +1,498 @@
...
+
+* `-pre`
+
+  Flag whose presence indicates the latest pre-release version of the library will be downloaded.
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: Dash? "prerelease" ... given Chicago.

Future tense and the sentence seems awkward ("whose presence") ...

> Obtain the latest prerelease version of the library.

File: aspnetcore/client-side/libman/libman-cli.md [hunk 10]
```diff
@@ -0,0 +1,498 @@
...
+
+* `--to <VERSION>`
+
+  The version number to which the library should be updated.
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: "to which" ...

> Specifies the version of the library.

File: aspnetcore/client-side/libman/libman-cli.md [hunk 11]
```diff
@@ -0,0 +1,498 @@
...
+  libman update jquery
+  ```
+
+* To update jQuery to version 3.3.1:
+
+  ```console
+  libman update jquery --to 3.3.1
+  ```
+
+* To update jQuery to the latest pre-release version:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: "pre-release" or "prerelease" ? ... given Chicago.

File: aspnetcore/client-side/libman/libman-cli.md [hunk 12]
```diff
@@ -0,0 +1,498 @@
...
+libman cache clean [<PROVIDER>] [--verbosity]
+libman cache list [--files] [--libraries] [--verbosity]
+libman cache [-h|--help]
+```
+
+### Arguments
+
+`PROVIDER`
+
+Used only with the `clean` command. The name of the provider whose cache is to be cleaned. Valid values include:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: Recommend "Only used with ..."

> The name of the provider whose cache is to be cleaned.

... perhaps ...

> Specifies the provider cache to clean.

File: aspnetcore/client-side/libman/libman-cli.md [hunk 13]
```diff
@@ -0,0 +1,498 @@
...
+
+[!INCLUDE [LibMan provider names](../../includes/libman-cli/provider-names.md)]
+
+### Options
+
+The following options can be appended to the `libman cache` command:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: > The following options are available for the \`libman cache\` command:

File: aspnetcore/client-side/libman/libman-cli.md [hunk 14]
```diff
@@ -0,0 +1,498 @@
...
+
+* `--libraries`
+
+  List the names of libraries that are cached.
+
+[!INCLUDE [standard-cli-options](../../includes/libman-cli/standard-cli-options.md)]
+
+### Examples
+
+* To view the names of cached libraries per provider, either of the following commands suffice:
```
Comment (guardrex) [inline aspnetcore/client-side/libman/libman-cli.md]: Localization? "work" over "suffice" ... hard to say.

**Comment by Rick-Anderson** [comment]
@justcla please review.

**Pull request merged**